chapters/hpmor-chapter-087.tex

```diff
@@ -60,7 +60,7 @@ \section{Thursday, April 16th, 1992}
 
 “I thought this seemed quite interesting,” her voice said.
 
-“Number fourteen, ‘Crozier’, true name unknown,” Harry read. “Wow, that is…that is the gaudiest chequered top hat I’ve ever seen. Wealth, at least six hundred thousand Galleons…so around thirty million pounds, not enough to make a Muggle famous, but good enough for the smaller wizard population, I guess. Rumoured to be a modern alias of the six-century-old Nicholas Flamel, the only known wizard to succeed at the incredibly difficult alchemical procedure for creating the Philosopher’s Stone, which enables the transmutation of base metals into gold or silver as well as…the Elixir of Life which indefinitely prolongs the youth and health of the user… Um, Hermione, this seems obviously false.”
+“Number fourteen, ‘Crozier’, true name unknown,” Harry read. “Wow, that is…that is the gaudiest chequered top hat I’ve ever seen. Wealth, at least six hundred thousand Galleons…so around thirty million pounds, not enough to make a Muggle famous, but good enough for the smaller wizard population, I guess. Rumoured to be a modern alias of the six-century-old Nicholas Flamel, the only known wizard to succeed at the incredibly difficult alchemical procedure for creating the Philosopher’s Stone, which enables the transmutation of base metals into gold or silver as well as…the Elixir of Life which indefinitely prolongs the youth and health of the user…Um, Hermione, this seems obviously false.”
```

Comment:
I think this is the end of a sentence, so add a full stop after the ellipsis rather than removing a space.